src/variablegrid.cpp

```diff
@@ -151,8 +151,8 @@ void VariableGridData::Init()
 			ConfigInfo* ci = m_cases[iCase]->GetConfiguration();
 			for (size_t iVarTable = 0; iVarTable < m_var_info_lookup_vec[iCase].size(); iVarTable++) {
 				// iterate over case vartables
-				wxString prepend = "";
-				if (ci->Technology.size() > 1) prepend = ci->Technology[iVarTable].Lower() + "_";
+				wxString prepend = ci->Technology[iVarTable].Lower() + "_";
```

Comment:
Think there's still an issue with some of the prepends having spaces in them, which is causing crashes when opening another case with Inputs Browser open: 

![image](https://github.com/NREL/SAM/assets/54449384/7802f4a2-0989-437c-a648-b95c5999cf48)

![image](https://github.com/NREL/SAM/assets/54449384/8519dd79-4e9c-468f-8278-d0324cc0a496)

Reply:
Issue seems consistent when you open another case with inputs browser open for these prepends with spaces. Might need to do a find and replace for " " or use different naming scheme.